open-scd.ts

```diff
@@ -276,7 +276,7 @@ export class OpenSCD extends Plugging(Editing(LitElement)) {
                 this.docName || nothing
               }" .doc=${this.doc} locale="${this.locale}" .docs=${
                 this.docs
-              }></${unsafeStatic(this.editor)}>`
+              } .editCount=${this.next}></${unsafeStatic(this.editor)}>`
```

Comment:
I think `this.next` should be renamed to `this.editCount` in order to keep with our pattern of having property names always be the same for `open-scd` and plugins.

Reply:
Agree 👍